verify_upload.py

```
@@ -0,0 +1,162 @@
+from pathlib import Path
+from typing import Any
+
+
+def load_json(filepath: Path) -> Any:
+    """Load JSON data from file."""
+    with open(filepath, encoding="utf-8") as f:
+        return json.load(f)
+
+
+def compare_resources(
+    transformed: list[dict], downloaded: list[dict], resource_type: str
```

Ruff (0.14.3)

Unused function argument: `resource_type`

(ARG001)